src/applications/ezr/tests/e2e/ezr-tera-information.cypress.spec.js

```diff
@@ -1,45 +1,61 @@
 import manifest from '../../manifest.json';
 import mockUser from './fixtures/mocks/mock-user';
+import mockUserAgentOrangeDob from './fixtures/mocks/tera/mock-user-agent-orange-dob';
+import mockUserCombatOperationsDob from './fixtures/mocks/tera/mock-user-combat-operations-dob';
+import mockUserOtherExposureDob from './fixtures/mocks/tera/mock-user-other-exposure-dob';
 import mockPrefill from './fixtures/mocks/mock-prefill.json';
+import mockPrefillAgentOrangeDob from './fixtures/mocks/tera/mock-prefill-agent-orange-dob.json';
+import mockPrefillCombatOperationsDob from './fixtures/mocks/tera/mock-prefill-combat-operations-dob.json';
+import mockPrefillOtherExposureDob from './fixtures/mocks/tera/mock-prefill-other-exposure-dob.json';
 import featureToggles from './fixtures/mocks/mock-features.json';
 import {
   goToNextPage,
   goToPreviousPage,
   fillGulfWarDateRange,
+  fillAgentOrangeDateRange,
+  fillTextWebComponent,
 } from './helpers';
 import { MOCK_ENROLLMENT_RESPONSE } from '../../utils/constants';
 import { advanceToHouseholdSection } from './helpers/household';
 
+function setUserData(user, prefillData) {
+  cy.login(user);
+  cy.intercept('GET', '/v0/feature_toggles*', featureToggles).as(
+    'mockFeatures',
+  );
+  cy.intercept('GET', '/v0/health_care_applications/enrollment_status*', {
+    statusCode: 200,
+    body: MOCK_ENROLLMENT_RESPONSE,
+  }).as('mockEnrollmentStatus');
+  cy.intercept('/v0/in_progress_forms/10-10EZR', {
+    statusCode: 200,
+    body: prefillData,
+  }).as('mockSip');
+}
+
+function goToToxicExposurePageAndCheckYes() {
+  cy.visit(manifest.rootUrl);
+  // Add '@mockfeatures' back to this list after removing the 'ezr_tera_branching_enabled' feature toggle
```

Comment:
Does this need added back still? Or is the comment out of date?

Reply:
It needed to be added back. It should be fixed now.